conference/user_panel.py

```diff
@@ -75,20 +75,25 @@ def manage_ticket(request, ticket_id):
     if ticket.user != request.user:
         return HttpResponse("Can't do", status=403)
 
-    ticket_configuration, _ = TicketConference.objects.get_or_create(ticket=ticket)
+    ticket_configuration, _ = TicketConference.objects.get_or_create(
+        ticket=ticket,
+        defaults={'name': ticket.name})
 
-    ticket_configuration_form = TicketConfigurationForm(
+    ticket_configuration_form = TicketConferenceConfigForm(
         instance=ticket_configuration, initial={"name": request.user.assopy_user.name()}
```

Comment:
I think if we use the `defaults` above we don't need to provide initial here, as it would be set to `ticket.name` already.

Reply:
Yes, once we run the script to fix all the old tickets. Right now it helps.